src/network/serverpackethandler.cpp

```diff
@@ -1150,8 +1150,8 @@ void Server::handleCommand_Interact(NetworkPacket *pkt)
 			if (pointed_object->isGone())
 				return;
 
-			ItemStack selected_item, hand_item;
-			ItemStack tool_item = playersao->getWieldedItem(&selected_item, &hand_item);
+			ItemStack selected_item;
+			ItemStack tool_item = playersao->getWieldedItem(&selected_item, nullptr);
```

Comment:
This will ignore custom `hand` tools (toolcaps). Intended?

Reply:
Oh right, providing `&hand_item` changes the behavior of `getWieldedItem` even if you leave the return unused doesn't it?

Reply:
Indeed, fixed.